Formula/corectl.rb

```diff
@@ -13,26 +13,64 @@ class Corectl < Formula
 
   depends_on "go" => :build
   depends_on "godep" => :build
+  depends_on "ocaml" => :build
+  depends_on "opam" => :build
   depends_on :macos => :yosemite
 
+  # until 0.7.1 is out
+  if build.stable?
```

Comment:
Please lob this into a `stable do` block with the stable url/sha256. We don't really do the `if build.stable?` syntax outside `def install` now.